config/initializers/customise_error.rb.rb

```diff
@@ -0,0 +1,3 @@
+ActionView::Base.field_error_proc = proc do |html_tag, _instance_tag|
```

Comment:
yes, classic trick to prevent rails from wrapping input fields with some extra error divs

Reply:
I'd prefer if we could avoid having to change a global Rails setting for this feature. I've come across difficult to debug bugs because of the use of this method.

Given that we haven't had to rely on this before can't we do the same for this feature?

Reply:
The problem would arise elsewhere, there aren't any other places where we match validation errors to a specific field.
Ultimately we could use https://github.com/ministryofjustice/govuk_elements_form_builder to ensure that GDS styling is enforced. But without this fix displaying a validation error on the matching input field would remain broken.